Data_Structures_Answers.md

```diff
@@ -22,9 +22,23 @@ We drop the + 2 and the space complexity is O(1)
 
 
 5. What is the runtime complexity of the provided code in `names.py`?
+The runtime complexity of the provided code in names.py is O(n^2).
+For each item in Names_1, the inner loop runs 10,000 times. Thus 10,000 * 10,000.
 
 6. What is the space complexity of the provided code in `names.py`?
+The space complexity of the code provided in names.py is O(n^2).
+It creates one array for each list of names.
```

Comment:
not quite! it's already using the array provided. look at the code again, you'll see there's no array being declared anywhere